benchmark/distinct_user_id/yyjson.h

```diff
@@ -6,7 +6,7 @@
 
 namespace distinct_user_id {
 
-class yyjson {
+struct yyjson {
 public:
```

Comment:
If you switch to a struct, you may not need the ugly public qualifier.

Reply:
That was the purpose :) Looks like I missed that one, thanks!